artifacts/attributes.adoc

```diff
@@ -67,3 +67,18 @@
 
 :LinkPluginsGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/configuring_plugins_in_red_hat_developer_hub/index
 :NameOfPluginsGuide: Configuring plugins in {product}
+
+:install-on-ocp-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_openshift_container_platform/index
+:install-on-ocp-book-title: Installing {product} on {ocp-short}
+
+:install-on-eks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_amazon_elastic_kubernetes_service/index
+:install-on-eks-book-title: Installing {product} on {eks-brand-name}
+
+:install-on-aks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_microsoft_azure_kubernetes_service/index
+:install-on-aks-book-title: Installing {product} on {aks-brand-name}
+
+:install-in-air-gap-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index
+:install-in-air-gap-book-title: Installing {product} in an air-gapped environment
+
+:upgrade-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/upgrading_red_hat_developer_hub/index
+:upgrade-book-title: Upgrading {product}
```

Comment:
Consider keeping the gerund and adding the Authentication and Authorization books:

```suggestion

:installing-on-ocp-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_openshift_container_platform/index
:installing-on-ocp-book-title: Installing {product} on {ocp-short}

:installing-on-eks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_amazon_elastic_kubernetes_service/index
:installing-on-eks-book-title: Installing {product} on {eks-brand-name}

:installing-on-aks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_microsoft_azure_kubernetes_service/index
:installing-on-aks-book-title: Installing {product} on {aks-brand-name}

:installing-in-air-gap-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index
:installing-in-air-gap-book-title: Installing {product} in an air-gapped environment

:authentication-book-url: https://docs.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/authentication/index
:authentication-book-title: Authentication

:authorization-book-url: https://docs.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/authorization/index
:authorization-book-title: Authorization

:upgrading-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/upgrading_red_hat_developer_hub/index
:upgrading-book-title: Upgrading {product}
```